phoenixRest/models/core/agenda_entry.py

```diff
@@ -31,21 +31,41 @@ class AgendaEntry(Base):
     event = relationship("Event")
     
     time = Column(DateTime, nullable=False)
+    newTime = Column(DateTime, nullable=False)
```

Comment:
Why is nullable `false` here but `true` in the migration?

Reply:
Fixed. All columns has been checked and matches eachother. nullable is no longer false in model, and true in migration.